docs/pages/versions/unversioned/sdk/facedetector.md

```diff
@@ -41,6 +41,8 @@ In order to configure detector's behavior modules pass a settings object which i
 - **mode? (_FaceDetector.Constants.Mode_)** -- Whether to detect faces in fast or accurate mode. Use `FaceDetector.Constants.Mode.{fast, accurate}`.
 - **detectLandmarks? (_FaceDetector.Constants.Landmarks_)** -- Whether to detect and return landmarks positions on the face (ears, eyes, mouth, cheeks, nose). Use `FaceDetector.Constants.Landmarks.{all, none}`.
 - **runClassifications? (_FaceDetector.Constants.Classifications_)** -- Whether to run additional classifications on detected faces (smiling probability, open eye probabilities). Use `FaceDetector.Constants.Classifications.{all, none}`.
+- **minDetectionInterval? (_long_)** -- Minimal interval in miliseconds between two face detection events being submitted to JS. Defaults to 0. Use, when you expect lots of faces for long time and are afraid of JS Bridge being overloaded.
+- **tracking? (_boolean_)** - Flag to enable tracking of faces between frames. If true, each face will be returned with `faceID` attribute which should be consistent across frames. Defaults to `false`;
```

Comment:
Should we use `faceID` or `faceId`? I think the latter is more common, are keys in JS dictionaries case-sensitive?

Reply:
Actually, there is a difference between SDK_32 and SDK_31, and since properties are case sensitive, it is a problem...
Will change to `faceId` and when backporting, will unify it.

Reply:
They are case-sensitive. Might be, that `faceId` is more common, but `faceID` is used in current versions, so I guess in order to keep it backward compatible, I will leave it as it is.